### PR TITLE
Logic

### PR DESCRIPTION
**Pull request by Urchmaney**

### CODE REVIEW

**Comment by shloch** [comment]
### PR under review by SHEY Louis CHIA

File: src/logics/operate.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    case '-':
+      result = Big(numberOne) - Big(numberTwo);
+      break;
+    case '*':
+      result = Big(numberOne) * Big(numberTwo);
+      break;
+    case '/':
+      result = Big(numberOne) / Big(numberTwo);
+      break;
+    case '%':
```
Comment (shloch) [inline src/logics/operate.js:18]: You forgot to manage division by ZERO 😊

**Review by shloch — APPROVED** [review]
### Hello @Urchmaney ,

Your work is good. I made an **optional** observation for you above which will be good for you to handle by maybe throwing an alert or which ever way creative you would handle it

On that reserve everything is perfect

You can merge this Pull Request (on to the next miletone)  👏 👏 👏

File: src/logics/operate.js (hunk 1 above)
Comment (Urchmaney) [reply src/logics/operate.js:18]: Hello I have a question.
Handling of divivsion by zero wont it come at the component level depending on what i recieve from the logic bit.

Comment (Urchmaney) [reply src/logics/operate.js:18]: Cause i cant(not suppose to) create alert in the logic section of the project.

Comment (shloch) [reply src/logics/operate.js:18]: @Urchmaney  Oh yeah totally bro, you can handle it there, was just drawing your attention

Comment (Urchmaney) [reply src/logics/operate.js:18]: ok thanks.

**Pull request merged**